client/src/styles/Header.css

```diff
@@ -32,7 +32,7 @@ nav {
 
 .linkNames {
   margin-left: 0rem;
-  color: #f9f8f7;
+  color: #a8a8a8;
```

Comment:
I would suggest starting a color palette if you do not already have one.

Reply:
should be coming soon!